packages/legend-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx

```diff
@@ -527,14 +527,15 @@ const SnowflakeDatasourceSpecificationEditor = observer(
           }
         />
         {/* TODO: we should reconsider adding this field, it's an optional boolean, should we default it to `undefined` when it's `false`?*/}
-        {/* <ConnectionEditor_BooleanEditor
+        <ConnectionEditor_BooleanEditor
           isReadOnly={isReadOnly}
           value={sourceSpec.quotedIdentifiersIgnoreCase}
-          propertyName={'cloud type'}
-          update={(value: string | undefined): void =>
-            sourceSpec.setCloudType(value)
+          propertyName={'quoted identifiers ignore case'}
+          description={'Set this when account has this flag set as true'}
```

Comment:
This description is a bit unclear to me. What do you think @MauricioUyaguari?

Reply:
For a bit more context, quoted Identifiers Ignore Case is the name of an actual flag that can be configured at the snowflake account/session level
https://docs.snowflake.com/en/sql-reference/identifiers-syntax.html#label-identifier-casing-parameter-howto

Reply:
Then should we try to use a summarized version of the description there as our description instead?

Reply:
@hardikmaheshwari also, maybe you have missed the comment I made, we should update our PR description with better context.

Reply:
Maybe something like `Treats alphabetic characters in double-quoted identifiers as uppercase`

Reply:
Actually, I not very sure here as to what would be easy for users.
I am not sure if users would really realize when they need to set this flag if we have a functional explanation in the description. We need this because alloy expects case-sensitive resultSet from db by default but this flag is an exception.

I am not saying that users modeling in studio would be database experts but I expect if we have this description they either check their db config or get details from their db experts

Reply:
@hardikmaheshwari I see what you mean, but instructing them with `Set this when the account has this flag set as true` is not that helpful either because if you're targeting non-expert, that description sounds rather cryptic to me. I think we should assume that we are making a form editor for folks who know what they are doing with their Snowflake connection.

That aside, in the future, I believe we have a plan to allow people to scan the spaces for already created connections, maybe this will help with non-expert users.

Reply:
I agree. 
Ok so to clarify setting this flag true/false does not make any difference to results from the user's side.
It's like a DB config which if set wrong, alloy won't be able to fetch results for the user.

So I believe, `Snowflake account config parameter` as the description makes sense to me.
I am still a bit resistant to the functional description `Treats alphabetic characters in double-quoted identifiers as uppercase` as this gives a feeling that setting this would change results.

Let me know your thoughts on this

Reply:
Except this is the description for a boolean field, maybe `Instructs Snowflake to treat alphabetic characters in double-quoted identifiers as uppercase`

Reply:
It seems this flag would be used by legend at session-level for services which is not the case.
Would give 1 last try at it (maybe we are overthinking this a bit as we can change the description with the user's feedback over time)

Reply:
@pierredebelen what do you think?

Reply:
Anyway, I'll merge this, but will change this to make it consistent with what we do in Studio in general. Will consult with @pierredebelen